install/page_dbsettings.php

```diff
@@ -167,9 +167,10 @@ function xoFormBlockCollation($name, $value, $label, $help = '', $link, $charset
 	$error = validateDbCharset ( $link, $vars ['DB_CHARSET'], $vars ['DB_COLLATION'] );
 	$db_exist = false;
 	if (empty ( $error )) {
-		if (! @mysql_select_db ( $vars ['DB_NAME'], $link )) {
+		if (! @mysqli_select_db ($link, $vars ['DB_NAME'] )) { // TODO update
```

Comment:
This will set the active db, or attempt to create a new db with that name if none exist, so if we specifically connect to a db by name above, this logic will be busted I think. Hence removing db name in the connection line.